R/prepare_data.R

```diff
@@ -1,3 +1,18 @@
+#' @title Check whether a package is installed
+#' @description A small internal function to check whether a required package
+#' is installed.
+#' This function is primarily intended to be used in mocking when testing for
+#' appropriate error messages from functions that rely on a package without
+#' that package being imported in {cfr}.
+#' The only current use case is [prepare_data.incidence2()].
+#' @param x A package name as a string.
+#' @return Quietly returns a logical for whether the package passed in `x` is
+#' available locally.
+#' @keywords internal
```

Comment:
Can optionally also add `@noRd` if you didn't want a file in the manual.

Reply:
Thanks, will do.

Reply:
Added in https://github.com/epiverse-trace/cfr/pull/100/commits/e877c4c72e1ab3b06fef7ff5367e6a273f03fca7